### PR TITLE
Upgrade v1.0.0

### PR DESCRIPTION
**Pull request by allanleal**
Checklist
* [x] Used a fork of the feedstock to propose changes
~* [ ] Bumped the build number (if the version is unchanged)~
* [x] Reset the build number to `0` (if the version changed)
~* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)~
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by nicoddemus** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,53 +1,52 @@
...
-    - boost
+    - boost=1.67
     - invoke
     - numpy 1.14.*
     - pybind11
     - python
     - tabulate
   run:
-    - boost
+    - boost=1.67
```
Comment (nicoddemus) [inline recipe/meta.yaml]: We should use conda-forge's pins here, so this pinning should be removed

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,53 +1,52 @@
...
 
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - cmake
+    - cmake>=3.13
     - ninja
   host:
-    - boost
+    - boost=1.67
```
Comment (nicoddemus) [inline recipe/meta.yaml]: We should use conda-forge's pins here, so this pinning should be removed

**Comment by allanleal** [comment]
We now have the following error for the win builds:
~~~
-- Found pybind11 v2.3.dev0: D:/bld/reaktoro_1550845694462/work/thirdparty/pybind11/include;D:/bld/reaktoro_1550845694462/_h_env/include
-- Configuring incomplete, errors occurred!
See also "D:/bld/reaktoro_1550845694462/work/build/CMakeFiles/CMakeOutput.log".
See also "D:/bld/reaktoro_1550845694462/work/build/CMakeFiles/CMakeError.log".
ninja: error: loading 'build.ninja': The system cannot find the file specified.
~~~

Strange.

**Pull request merged**